test/unit_tests/braket/default_simulator/test_density_matrix_simulator.py

```diff
@@ -795,3 +795,42 @@ def test_measure_targets():
     assert 400 < np.sum(measurements, axis=0)[0] < 600
     assert len(measurements[0]) == 1
     assert result.measuredQubits == [0]
+
+
+def test_measure_no_gates():
+    qasm = """
+    bit[4] b;
+    qubit[4] q;
+    b[0] = measure q[0];
+    b[1] = measure q[1];
+    b[2] = measure q[2];
+    b[3] = measure q[3];
```

Comment:
since we know the number of qubits (i.e. we have `qubit[4] q;`), can we do `b = measure q;` ? I assume the 4 is because there's gates/measurements on 4 qubits, so the BDK does know that information.

Reply:
We can do `b = measure q`. I can update this test

Reply:
The BDK would be able to know the number of qubits on the circuit based on the gates/measurements added. 
The measurement targets have to be specified though.